### PR TITLE
Add analytics token

### PR DESCRIPTION
**Pull request by adimiz1**
### Brief Summary of Changes
Add React Native analytics token

#### What does this PR address?
- [ ] GitHub issue (Add reference - #XX)
- [ ] Refactoring
- [x] New feature
- [ ] Bug fix
- [ ] Adds more tests

#### Are tests included?
- [x] Yes
- [ ] No

#### Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

File: src/internal/SDKAnalyticsConstants.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    const version = Platform.Version;
+    return version.toString();
+  } catch {
+    return "0.0.0"
+  }
+}
+
+const getSDKVersion = () => {
+  try{
+    const SDKVersionPackageJson = require('cloudinary-react-native/package.json')
```
Comment (lbendavid-cloudinary) [inline src/internal/SDKAnalyticsConstants.ts:14]: Don't we want to have a script that populates this SDK version inside `SDKAnalyticsConstants` (like we have in other FE SDK's)

**Review by lbendavid-cloudinary — APPROVED** [review]

File: src/internal/SDKAnalyticsConstants.ts (hunk 1 above)
Comment (adimiz1) [reply src/internal/SDKAnalyticsConstants.ts:14]: I'll check, though I'm not sure why a script is required here

Comment (lbendavid-cloudinary) [reply src/internal/SDKAnalyticsConstants.ts:14]: Maybe the idea behind it was to not load `package.json` on the customer's side... but i cannot think of other good reason to do that

**Review by const-cloudinary — APPROVED** [review]

File: src/internal/SDKAnalyticsConstants.ts (hunk 1 above)
Comment (adimiz1) [reply src/internal/SDKAnalyticsConstants.ts:14]: I could not find a justification to create a script here, unlike the FEF we only have 1 package here who make use of extracting the version, if we find there's a need for a script we can write one in the future.

**Pull request merged**